virtcontainers/device/config/config.go

```diff
@@ -75,6 +80,12 @@ var SysDevPrefix = "/sys/dev"
 // SysIOMMUPath is static string of /sys/kernel/iommu_groups
 var SysIOMMUPath = "/sys/kernel/iommu_groups"
 
+// VhostUserSockDir is static string
```

Comment:
This comment (and the one below) don't really describe what the variable is for - it's obvious they are static strings, but what are the strings for? :)